### DISCUSSION TITLE
Metatag classname

### DISCUSSION THREAD

**Discussion by ihor-zinchenko** [vercel/next.js#55371] [Help]
### Summary

I need to add classname for meta tags (elastic)
It should be like that
`<meta class="elastic" name="type" content="Some type">`

`
export async function generateMetadata({ params }): Promise<Metadata> {
  const attributes = await preloadData(params);
  return {
    title: `Some title`,
    description: 'Description',
    other: {
      type: 'Some type'
    }
  };
}
`

Is it possible?

**Comment by icyJoseph** [comment]
I don't think that's entirely possible with their API. Why do you need that?

**Reply by ihor-zinchenko** [reply]
Its for Elastic. Elastic searching metatags via classname.
